<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserWindow</class>
 <widget class="QMainWindow" name="UserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1560</width>
    <height>940</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1560</width>
    <height>940</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1560</width>
    <height>940</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>User work window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pbExit">
    <property name="geometry">
     <rect>
      <x>1400</x>
      <y>830</y>
      <width>144</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>144</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>144</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leOwnId">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>100</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>28</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>28</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(247, 247, 247);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="leOwnLogin">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>200</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>28</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>28</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(247, 247, 247);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelOwnId">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>28</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>28</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Own Id</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOwnLogin">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>120</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>28</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Own Login</string>
    </property>
   </widget>
   <widget class="QTableView" name="tvTasks">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>1520</width>
      <height>410</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>1520</width>
      <height>410</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1520</width>
      <height>410</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#tvTasks {
	border: 1px solid rgb(180, 180, 180);
    border-radius: 1px;
	border-color: #777777;
	show-decoration-selected: 0;
}

#tvTasks::item {
  border : 0px;
  padding-left: 10px;
}

#tvTasks::item:selected {
	color: black;
	background-color: #FFF3BF;
}

#tvTasks::branch:selected { 
	background: #FFF3BF; 
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTasks">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>120</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>28</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tasks</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>590</y>
      <width>1521</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pbGetTasks">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>280</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>280</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>280</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Require Tasks List</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkbOnlyOwn">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>530</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>160</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>160</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Only own</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelStatus">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>720</y>
      <width>120</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>28</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>28</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Task Status</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbTaskStatus">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>770</y>
      <width>200</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>28</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>28</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="leDescription">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>670</y>
      <width>1400</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>1400</width>
      <height>28</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1400</width>
      <height>28</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="labelDate">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>620</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>28</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>28</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Deadline</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leDeadLine">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>620</y>
      <width>150</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>28</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>28</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="leTaskName">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>620</y>
      <width>300</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>28</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>28</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="labelDescription">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>670</y>
      <width>100</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>28</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>28</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Description</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTaskName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>620</y>
      <width>100</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>28</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>28</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Task Name</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>710</y>
      <width>460</width>
      <height>140</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>460</width>
      <height>140</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>460</width>
      <height>140</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Action Type</string>
    </property>
    <widget class="QRadioButton" name="rbTakeTask">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>200</width>
       <height>28</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>28</height>
      </size>
     </property>
     <property name="text">
      <string>Take Task</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbChangeStatus">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>200</width>
       <height>28</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>28</height>
      </size>
     </property>
     <property name="text">
      <string>Change Status</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbSetDeadline">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>100</y>
       <width>200</width>
       <height>28</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>28</height>
      </size>
     </property>
     <property name="text">
      <string>Set Deadline</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbCreateTask">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>200</width>
       <height>28</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>28</height>
      </size>
     </property>
     <property name="text">
      <string>Create task</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pbApply">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>720</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbClear">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>720</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Clear fields</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelUser">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>620</y>
      <width>140</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>140</width>
      <height>28</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>140</width>
      <height>28</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Appointed to</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leFullname">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>620</y>
      <width>630</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>630</width>
      <height>28</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>630</width>
      <height>28</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(247, 247, 247);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelOwnFullname">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>20</y>
      <width>140</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>140</width>
      <height>28</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>140</width>
      <height>28</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Own Fullname</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leOwnFullname">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>20</y>
      <width>630</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>630</width>
      <height>28</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>630</width>
      <height>28</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(247, 247, 247);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1560</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
